java/Brewfile

```diff
@@ -17,3 +17,4 @@
 
 brew "openjdk@11"
 brew "sccache"
+brew "curl"
```

Comment:
Perhaps keep this file in alphabetical order?

Reply:
Juste deleted, that was added for testing purpose that at the end we add CURL_ROOT on the github.yml file